### PR TITLE
test: migrate njk leftovers

### PR DESCRIPTION
**Pull request by droshev**
Migrate njk/backstop leftovertests

@jbadan shall we delete the whole backstop configuration, folders, etc. ?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "style:compile": "node-sass -q --output-style expanded --precision 5 src/ --output dist/",
     "style:postCSS": "postcss --config config/postcss.config.js  --replace dist/*.css",
     "style:remove": "if [ -d dist/ ]; then rm -rf dist/; fi",
-    "test": "npm run test:linux-fix && server-test start:playground http-get://localhost:3030 test:visual",
+    "test": "npm run storybook:visual",
     "test:linux-fix": "cp -rf scripts/runDocker.js node_modules/backstopjs/core/util/",
```
Comment (jbadan) [inline package.json:32]: Can we replace all these test commands with the storybook testing ones and delete the storybook testing ones to clean this up a bit?

Comment (droshev) [reply package.json:32]: should i do it in this PR?

Comment (droshev) [reply package.json:31]: done

File: .storybook/custom/custom.scss [hunk 2]
```diff
@@ -308,3 +308,28 @@ body.sb-show-main {
   color: #32363a;
   color: var(--sapTextColor, #32363a);
```
Comment (jbadan) [inline .storybook/custom/custom.scss:309]: Why aren't we using the theme for some of this: https://github.com/SAP/fundamental-styles/blob/master/.storybook/custom/fundamentals.js#L4

File: .storybook/custom/custom.scss [hunk 3]
```diff
@@ -308,3 +308,28 @@ body.sb-show-main {
...
+  background: gold;
+  font-size: 14px;
+  box-shadow: 0 5px 20px 0 blue;
+}
+
+.fddocs.fd-container {
+  min-height: 20px;
+  padding: 20px;
+  background: lightblue;
+  border: 1px solid yellow;
```
Comment (jbadan) [inline .storybook/custom/custom.scss:326]: Why yellow 😆

File: .travis.yml [hunk 4]
```diff
@@ -11,9 +11,7 @@ jobs:
   include:
   - stage: Lint, Test and Size
     script:
-      - docker pull backstopjs/backstopjs:3.9.2
-      - npm run test:ci-config
-      - npm run test
+      - npm run test:ci
```
Comment (jbadan) [inline .travis.yml:14]: Does this do anything besides run storybook?

File: .storybook/custom/custom.scss (hunk 3 above)
Comment (jbadan) [reply .storybook/custom/custom.scss:326]: oh I see it's for the examples - it feels weird to use a class like this - why not a docs-container?

**Pull request merged**